apps/emqx/test/emqx_cm_SUITE.erl

```diff
@@ -250,7 +250,7 @@ t_kick_session(_) ->
 
             timer:sleep(1000)
         end),
-    ct:sleep(100),
```

Comment:
please keep the sleep for now.
this case has been removed in another PR: https://github.com/emqx/emqx/pull/6113